addons/pos_self_order/static/src/kiosk/components/cancel_popup/cancel_popup.scss

```diff
@@ -0,0 +1,5 @@
+.o_kiosk-cancel_popup {
+    .modal-dialog {
+        top: 40%;
```

Comment:
The height should be replaced to auto too. For the moment, it is 100% due to modal-dialog class. This lead to the fact that the page can be scrolled when the popup is there and thus the popup being move to the top of the screen.